components/FilterBar/index.jsx

```diff
@@ -94,14 +94,28 @@ class FilterBar extends Component {
       return;
     }
 
-    const height = this.element.offsetHeight;
-
-    if (height > 0 && height !== this.state.spacerHeight) {
-      this.setState({ spacerHeight: height });
-    }
+    /**
+     * Wait for the next tick to be sure the component has updated when
+     * this is triggered from the child components.
+     */
+    setTimeout(() => {
+      const height = this.element.offsetHeight;
+
+      if (height > 0 && height !== this.state.spacerHeight) {
+        this.setState({ spacerHeight: height });
+      }
+    }, 0);
```

Comment:
Unfortunately this doesn't work perfectly. I still can reproduce the issue - especially on a device. I've played around with the timeout and values between 150 and 200 seemed to work, but playing with timeouts isn't really reliable :-( Additionally the transitions on height changes get visible with this approach.